tests/framework/extensions/ingresses/ingresses.go

```diff
@@ -19,3 +32,49 @@ func AccessIngressExternally(client *rancher.Client, hostname string, isWithTLS
 
 	return result.Ok, err
 }
+
+// CreateIngress will create an Ingress object in the downstream cluster.
+func CreateIngress(client *v1.Client, ingressName string, serviceName string) (*v1.SteveAPIObject, error) {
```

Comment:
Can we instead make this function take the templates rather than them being set within the function? So there is more flexibility in terms of how this function is created.

Reply:
Spoke offline, addressed this.